bvm/ballerina-runtime/src/main/java/io/ballerina/runtime/internal/launch/LaunchUtils.java

```diff
@@ -58,6 +59,7 @@
 public class LaunchUtils {
 
     private static final PrintStream outStream = System.out;
+    private static final Pattern UNESCAPED_NEWLINE_CHAR = Pattern.compile("\\\\n(?=(?:[^\"]*\"[^\"]*\")*[^\"]*$)");
```

Comment:
It seems windows regex pattern needs to have additional `\r` as well.

Reply:
yes. We need to use a common pattern for all platforms. 
I'll check and update this.